manifests/register.pp

```diff
@@ -0,0 +1,26 @@
+# @summary Register the host as foreman
```

Comment:
```suggestion
# @summary Register the host as Foreman
```

Nit across all of the files in here, Foreman is a proper noun so we should strive to treat it that way in our writing.